Link to our RubyDoc.info docs from the README (#26)
duffelhq/duffel-api-ruby: README.md

# Duffel API Ruby client library

> [![RubyDoc.info documentation](http://img.shields.io/badge/yard-docs-blue.svg)](https://rubydoc.info/github/duffelhq/duffel-api-ruby/main)

Reviewer: Is the branch name needed in the URL? Not a blocker, just seemed odd.
```suggestion
[![RubyDoc.info documentation](http://img.shields.io/badge/yard-docs-blue.svg)](https://rubydoc.info/github/duffelhq/duffel-api-ruby)
```

end
```

## Learn more

> You can find complete documentation on the library's classes and methods in the in-code
> documentation, [published to RubyDoc.info](https://rubydoc.info/github/duffelhq/duffel-api-ruby/main).

Reviewer: Small non-blocking suggestion.
```suggestion
You can find complete documentation on this library's classes and methods in the in-code
documentation on [RubyDoc.info](https://rubydoc.info/github/duffelhq/duffel-api-ruby).
```
